helm-chart/index.yaml

```diff
@@ -22,20 +22,40 @@
 apiVersion: v1
 entries:
   openj9-jitserver-chart:
+  - apiVersion: v2
+    appVersion: 0.27.0
+    created: "2021-08-17T20:37:10.271808346-04:00"
+    description: "Eclipse OpenJ9 JITServer Helm Chart. \n\nLicense\n\nThis chart is
+      made available under the terms of the Eclipse Public License 2.0 \nwhich accompanies
+      this distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+      \nor the Apache License, Version 2.0 which accompanies this distribution and
+      \nis available at https://www.apache.org/licenses/LICENSE-2.0.\n\nThe JDK binaries
+      installed by this chart are licensed under the GPLv2+CE."
+    digest: c4b2e28107027178ff22ed67eaae8cdbf3eed03f6f102902466b2c1d339b859f
+    icon: https://pbs.twimg.com/profile_images/910853900181364736/bmWkKoLn_400x400.jpg
+    keywords:
+    - amd64
+    - ppc64le
+    - Java
+    - Eclipse
+    - OpenJ9
+    - JITServer
+    - JVM
+    - JIT
+    name: openj9-jitserver-chart
+    type: application
+    urls:
+    - https://github.com/eclipse-openj9/openj9-utils/files/7002653/openj9-jitserver-chart-0.27.0.tar.gz/openj9-jitserver-chart-0.27.0.tgz
```

Comment:
I would keep the url : 
```
https://github.com/eclipse-openj9/openj9-utils/files/7002653/openj9-jitserver-chart-0.27.0.tar.gz
```
instead of 
```
https://github.com/eclipse-openj9/openj9-utils/files/7002653/openj9-jitserver-chart-0.27.0.tar.gz/openj9-jitserver-chart-0.27.0.tgz
```

Reply:
Good catch. I didn't observe that helm changed the url I gave on the command line.
Done.